.github/workflows/ena-submission-image.yaml

```diff
@@ -98,3 +98,9 @@ jobs:
           for TAG in "${TAGS[@]}"; do
             docker buildx imagetools create --tag $TAG ${{ env.DOCKER_IMAGE_NAME }}:${{ env.DIR_HASH }}
           done
+      # Run smoke test to ensure the image is functional
```

Comment:
is this required? we will see it in the unit tests

Reply:
If you mean the comment no. Tests are never required - this is an easy way to check that the docker image works as expected. We don't run the docker image in any actual tests in contrast to most others which run in previews and E2E/integration so I felt adding a simple check here helps us ensure we notice if we break the image early.

The only downside is:
- extra 20s runtime (but not delay to image being pushed) [basically nothing]
- if unit tests fail, so does the docker image workflow, but the image still gets pushed, as long as you know it runs unit tests in docker as well this has no downside.

Reply:
Makes sense, but it's normal to run basic tests on packaging artifacts, these are not expensive :)